### PR TITLE
Active ride bottom sheet

### PR DESCRIPTION
**Pull request by avivbarniv**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/Components/DriverCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+interface DriverCardProps {
+    activeRide: boolean;
+    ride: RideInterface;
+}
+
+const formatDriverRating = (rating: number): string | 0 => rating && rating.toFixed(1);
+const MAX_NAME_LENGTH = 20;
+
+const formatDriverName = (name: string): string => name && name.substring(0, MAX_NAME_LENGTH);
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/Components/DriverCard/index.tsx:27]: wouldn't it be better do do an ellipses or something?

File: examples/client/Locomotion/src/context/newRideContext/index.tsx [hunk 2]
```diff
@@ -116,13 +130,21 @@ const RidePageContextProvider = ({ children }: {
...
   const [serviceEstimations, setServiceEstimations] = useState<any | null>(null);
-  const [ride, setRide] = useState<Ride>({});
+  const [ride, setRide] = useState<RideInterface>({});
   const [chosenService, setChosenService] = useState<any | null>(null);
   const [lastSelectedLocation, saveSelectedLocation] = useState(false);
   const [rideRequestLoading, setRideRequestLoading] = useState(false);
   const [serviceRequestFailed, setServiceRequestFailed] = useState<boolean>(false);
   const intervalRef = useRef<any>();
 
+  // const serviceType = ride.serviceType || await rideApi.getService(ride.serviceTypeId)
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/context/newRideContext/index.tsx:140]: needed?

**Review by lieblichaaron — APPROVED** [review]

File: examples/client/Locomotion/src/context/newRideContext/index.tsx (hunk 2 above)
Comment (avivbarniv) [reply examples/client/Locomotion/src/context/newRideContext/index.tsx:140]: Yes, for future use

File: examples/client/Locomotion/src/Components/DriverCard/index.tsx (hunk 1 above)
Comment (avivbarniv) [reply examples/client/Locomotion/src/Components/DriverCard/index.tsx:27]: Agree, but I just copied the code

**Pull request merged**